### PR TITLE
fix(abbreviations.json): update norm symbol

### PR DESCRIPTION
**Pull request by fpvandoorn**
* Change the symbol associated to `\||` and `\fuzzy`
* `\parallel` gave the same symbol as `\||` before, so we don't change that one
* Remove `\shortparallel` which was the same as the previous `\||`
* Remove `\nshortparallel` and `\||n` which are the same as `\nparallel`
* Related to leanprover-community/mathlib#17575

### CODE REVIEW

File: src/abbreviation/abbreviations.json [hunk 1]
```diff
@@ -909,7 +907,8 @@
     "||=": "⊫",
     "|||-": "⊪",
     "||n": "∦",
```
Comment (eric-wieser) [inline src/abbreviation/abbreviations.json:909]: I think we should remove this one and rename it to `paralleln` or `nparallel`

Comment (fpvandoorn) [reply src/abbreviation/abbreviations.json:909]: `nparallel` already exists for this symbol; I removed this one.

**Comment by sgouezel** [comment]
bors r+
Thanks!

**Comment by sgouezel** [comment]
(maybe there's no bors here...)

**Comment by eric-wieser** [comment]
Nope, this needs a manual merge by @gebner (and of course a release for it to make any difference)

**Pull request merged**